augur/tasks/git/util/facade_worker/facade_worker/analyzecommit.py

```diff
@@ -126,68 +126,28 @@ def store_commit(repos_id,commit,filename,
 		#session.logger.info(f"Timestamp: {author_timestamp}")
 		commit_record = {
 			'repo_id' : repos_id,
-			'commit' : str(commit),
```

Comment:
Am I right that this is making everything map directly, allowing us to remove mapping code below?

Reply:
Yes mapping is not needed since I just use the table field names in this branch.